### ISSUE TITLE
Remove compilation dependencies pointing to SQLite, etc.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Benknightdark/elmah#113]
```
What new or enhanced feature are you proposing?

If possible, remove compilation dependencies for "Elmah.dll" such that the 
file (1) "System.Data.SQLite.dll" and the file (2) "VistaDB.NET20.dll" are 
not pulled into a target ASP.NET web site project when there is a file-
based reference in the target ASP.NET web site project pointing 
to "Elmah.dll" and one compiles the project in full. Note that it (1) and 
(2) get pulled an copied into the ASP.NET web site's Bin directory when 
one compiles but it appears that if one removes (1) and (2) manually then 
Elmah continues to work fine (when not using VistaDb or SqLite of course).

What goal would this enhancement help you achieve?

This would make the project a bit cleaner, more lightweight, less 
complicated, easier to use, and easier to deploy. IMHO.

```

Original issue reported on code.google.com by `mkamo...@gmail.com` on 2 Jun 2009 at 12:41

**Comment by GoogleCodeExporter** [comment]
Original comment by `azizatif` on 8 Jun 2009 at 12:41
- Changed title: **Remove compilation dependencies pointing to SQLite and VistaDB**
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `azizatif` on 29 Sep 2009 at 6:21
- Added labels: **Milestone-Release1.2**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Consider investigating TypeForwardTo attribute to make the migration easier:
http://tinyurl.com/TypeForwardToAttribute

```

Original comment by `azizatif` on 29 Sep 2009 at 6:31

**Comment by GoogleCodeExporter** [comment]
Original comment by `azizatif` on 29 Sep 2009 at 6:32
- Changed title: **Remove compilation dependencies pointing to SQLite, etc.**

**Comment by GoogleCodeExporter** [comment]
```
Addressed somewhat by NuGet packages[1] for now.

[1] 
http://nuget.org/List/Search?packageType=Packages&searchTerm=elmah&sortOrder=pac
kage-download-count
```

Original comment by `azizatif` on 5 May 2011 at 9:59
- Removed labels: **Milestone-Release1.2**

**Comment by GoogleCodeExporter** [comment]
```
Issue 298 has been merged into this issue.
```

Original comment by `azizatif` on 26 Jun 2012 at 6:56

**Comment by GoogleCodeExporter** [comment]
Original comment by `azizatif` on 18 Dec 2013 at 5:19
- Changed state: **Started**
- Added labels: **Milestone-Release2.0**

**Comment by GoogleCodeExporter** [comment]
```
Issue 313 has been merged into this issue.
```

Original comment by `azizatif` on 18 Dec 2013 at 5:25

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 938bc80cf3f7.
```

Original comment by `azizatif` on 18 Dec 2013 at 5:37
- Changed state: **Fixed**
